da/mock/mock.go

```diff
@@ -68,7 +69,7 @@ func (m *DataAvailabilityLayerClient) Stop() error {
 // SubmitBlock submits the passed in block to the DA layer.
 // This should create a transaction which (potentially)
 // triggers a state transition in the DA layer.
-func (m *DataAvailabilityLayerClient) SubmitBlock(block *types.Block) da.ResultSubmitBlock {
+func (m *DataAvailabilityLayerClient) SubmitBlock(context context.Context, block *types.Block) da.ResultSubmitBlock {
```

Comment:
Same as above, quick change from `context` to `ctx`, as well for `CheckBlockAvailability` and `RetrieveBlocks`.